### ISSUE TITLE
question about param _n_labels

### ISSUE THREAD

**Issue by HUSTHY** [dwadden/dygiepp#8]
why the param in code  TimeDistributed(torch.nn.Linear(mention_feedforward.get_output_dim(), self._n_labels - 1)) is self._n_labels - 1 , should it be self._n_labels ?
And on the other hand,why do you design dummy_scores in ner.py ?

**Comment by dwadden** [comment]
I added some comments to address your question. Basically, the null label is always given a score of 0.
https://github.com/dwadden/dygiepp/commit/b3a2a5f526afe350a03c9d85ba4c1b8e2ccdbc89
